Tests/LibWeb/Text/input/wpt-import/interfaces/credential-management.idl

```diff
@@ -0,0 +1,107 @@
+// GENERATED CONTENT - DO NOT EDIT
+// Content was automatically extracted by Reffy into webref
+// (https://github.com/w3c/webref)
+// Source: Credential Management Level 1 (https://w3c.github.io/webappsec-credential-management/)
+
+[Exposed=Window, SecureContext]
```

Comment:
@tcl3 Do we have any other "idlharness" tests that are imported? I can't tell just looking if we need these idl files commited into our repo or not.